device-types/HPE/Aruba-EC-XL-H.yaml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: HPE
+model: Aruba EC-XL-H
+slug: hpe-aruba-ec-xl-h
+part_number: '201756'
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight: 33.5
+weight_unit: lb
+front_image: true
+rear_image: true
+console-ports:
+  - name: console
+    type: de-9
+    label: console
+power-ports:
```

Comment:
Hi Thank you for your contribution.
Are these modular PSUs? If so they should be modeled as `module-bays` with the required PSUs added to the modules folder if they don't already exist.